packages/calcite-components/src/components/block/block.e2e.ts

```diff
@@ -186,26 +186,25 @@ describe("calcite-block", () => {
 
   it("can display/hide content", async () => {
     const page = await newE2EPage();
-    await page.setContent("<calcite-block><div>some content</div></calcite-block>");
-    let element = await page.find("calcite-block");
-    let content = await page.find(`calcite-block >>> .${CSS.content}`);
+    await page.setContent(
+      html`<calcite-block heading="heading" description="description"><div>Hello world!</div></calcite-block>`,
+    );
+    await skipAnimations(page);
+    await page.waitForChanges();
 
+    const element = await page.find("calcite-block");
+    const content = await page.find(`calcite-block >>> section.${CSS.content}`);
```

Comment:
I found something weird here. When using the `>>>` piercing selector, it was finding some other `CSS.content` class that is a div. Adding `section` in front of this fixed it but Its also concerning it was finding this element which it shouldn't find. Using an id is fine too.

Reply:
I agree it can be confusing, but this is working as expected (see https://stenciljs.com/docs/end-to-end-testing#find-an-element-in-the-shadow-dom). This also came up in a [Stencil issue](https://github.com/ionic-team/stencil/issues/5717).

Reply:
Interesting. I didn't know it searched all shadow roots. good to know!